src/GitPrompt.ps1

```diff
@@ -13,15 +13,19 @@ if ($Host.UI.RawUI.BackgroundColor -eq [ConsoleColor]::DarkMagenta) {
     $s.WorkingColor.ForegroundColor             = 'Red'
 }
 
-$isAdminProcess = Test-Administrator
-$adminHeader = if ($isAdminProcess) { 'Administrator: ' } else { '' }
+$IsAdmin = Test-Administrator
 
-$WindowTitleSupported = $true
-# TODO: Hmm, this is a curious way to detemine window title supported
-# Could do $host.Name -eq "Package Manager Host" but that is kinda specific
-# Could attempt to change it and catch any exception and then set this to $false
-if (Get-Module NuGet) {
-    $WindowTitleSupported = $false
+# Probe $Host.UI.RawUI.WindowTitle to see if it can be set without errors
+$WindowTitleSupported = $false
+try {
+    $global:PreviousWindowTitle = $Host.UI.RawUI.WindowTitle
+    $newTitle = "$origTitle "
```

Comment:
Should this be `"$PreviousWindowTitle "`?

Reply:
Doh!  Yes it should be. Thanks.  I'll fix it tonight.